feat: Add `backgroundColor` property

michaelbukachi opened the pull request:
As discussed in issue #11 , This pull requests adds the `backgroundColor` property to the `WhatsNew` class. This will enable devs to use normal Color integers e.g
```
val whatsNew = WhatsNew.newInstance()
whatsNew.backgroundColor = ContextCompat.getColor(this, R.color.colorAccent)
```
or
```
val whatsNew = WhatsNew.newInstance()
whatsNew.backgroundColor = Color.parseColor("#FF0000")
```

**Note**: I've set the `backgroundColor` property to have priority over `backgroundColorResource`. So in a situation where both are specified, the `backgroundColor` property will be used.

TonnyL commented:
Your pr has conflicts.

michaelbukachi commented:
@TonnyL What files are conflicting?

TonnyL commented:
<img width="1028" alt="Screen Shot 2019-04-24 at 7 55 02 PM" src="https://user-images.githubusercontent.com/13329148/56657540-dedd9000-66ca-11e9-8307-743d70629678.png">

Your second commit has something wrong.

michaelbukachi commented:
Hmm. That's odd. I'm actually up to date with your repo already. Can you see which specific files are conflicting?

TonnyL commented:
<img width="685" alt="Screen Shot 2019-04-24 at 10 16 56 PM" src="https://user-images.githubusercontent.com/13329148/56666602-d2fbc900-66de-11e9-9506-467db97eb461.png">

Use this quick way to sync my repo. Here is the conflicted commit.

<img width="2550" alt="Screen Shot 2019-04-24 at 10 19 41 PM" src="https://user-images.githubusercontent.com/13329148/56666851-3ede3180-66df-11e9-9eb6-0fb14e596a92.png">

michaelbukachi commented:
@TonnyL Can you try now?

TonnyL commented:
> @TonnyL Can you try now?

stays the same.

michaelbukachi commented:
Hello @TonnyL can you check now?

TonnyL commented:
> Hello @TonnyL can you check now?

stays the same.

michaelbukachi committed:
build:  Download android gradle plugin

This commit downgrades the android gradle plugin from `3.5.0-alpha04` to `3.3.2` to improve compatibility with older android studio versions.

michaelbukachi committed:
feat: Add `backgroundColor` property

This commit adds the `backgroundColor` property to the WhatsNew class. This will enable devs to use normal color integers e.g
```
val whatsNew = WhatsNew.newInstance()
whatsNew.backgroundColor = ContextCompat.getColor(this, R.color.colorAccent)
```
or
```
val whatsNew = WhatsNew.newInstance()
whatsNew.backgroundColor = Color.parseColor("#FF0000")
```

michaelbukachi commented:
Hello @TonnyL 
So I cloned the repo again and did the updates again. Can you check now?

TonnyL commented:
> Hello @TonnyL
> So I cloned the repo again and did the updates again. Can you check now?

It's ok now. Will review later.
